tsconfig.json

```diff
@@ -14,11 +14,15 @@
 		"removeComments": true,
 		"sourceMap": true,
 
-		"outDir": "../dist/server/",
-		"paths": {}
+		"outDir": "./dist/",
+		"paths": {},
+		"allowJs": true
 	},
 	"include": [
-		"./**/*"
+		"./server/**/*",
+		"**/*.mjs"
 	],
-	"exclude": []
+	"exclude": [
+		"./client"
+	]
```

Comment:
Why to you moved the server tsconfig file?
Server and Client configuration were separate. Moving the server tsconfig in the root directory adds confusion (IMHO).